dbinspector/src/main/kotlin/im/dino/dbinspector/domain/shared/models/dsl/query.kt

```diff
@@ -0,0 +1,21 @@
+package im.dino.dbinspector.domain.shared.models.dsl
```

Comment:
Why the lowercase filename? 🤔

Reply:
@JonatanPlesko Because it's an unwritten rule for file that don't hold any classes. Like this one.